### ISSUE TITLE
Italicizing non-literal syntax?

### ISSUE THREAD

**Issue by paroche** [javascript-tutorial/en.javascript.info#1500]
How about, when showing syntax like:

node.append(...nodes or strings) – append nodes or strings at the end of node,
node.prepend(...nodes or strings) – insert nodes or strings into the beginning of node,

that "node" (in this case), or whatever syntax is not literal, that is, that stands for some general type of thing, be put in italics, i.e., in this case, have instead:

_node_.append(...nodes or strings) – append nodes or strings at the end of node,
_node_.prepend(...nodes or strings) – insert nodes or strings into the beginning of node,

(From: [https://javascript.info/modifying-document](url))

I think that would make the meaning of the syntax clearer. Also, this is what MSN does in their Syntax descriptions, e.g.:
_element_.appendChild(aChild);
[https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild](url)

I think this would also address some of my earlier concerns with the use of "str", "array", "obj", etc. from a couple months ago.

**Comment by paroche** [comment]
Or does that not work when _`entering code`_?

**Comment by iliakan** [comment]
Indeed, making things italic isn't a very good idea, because it doesn't look good.

Is there a real need and confusion here?

**Comment by paroche** [comment]
I don't see why it has to not look good. It doesn't seem to work with backticks, but in regular HTML, having, for instance, `<code><var>elem</var>.getElementById</code>`, looks like this:

<code><var>elem</var>.getElementById</code>

Which I think looks fine, and is also the way, for instance, Mozilla represents such things on their MDN web docs site.

And yes, I think it really helps to have the variable/shorthand portion of the code easily distinguished from the actual syntax part, especially when the syntax is being encountered for the first time and/or the reader is not accustomed to the conventions being used for variables and shorthand expressions; and not having that **can** lead to confusion. This was, for instance, the case when I was having issues some time back with the use of 'var', 'arr', 'obj', etc. in code examples. I figured it out, but it slowed me down.

**Comment by Zearin** [comment]
I strongly agree with [the previous comment](#issuecomment-549700127).  I think `<var>` is one of the most under-used elements in HTML, and it is incredibly helpful for cases like the above.

Furthermore, although I _also_ think italics are fine, they are just a default styling.  Using `<var>` means you can style it with CSS like any other element.  You can still use `<var>` to distinguish the “non-literal” syntax from its surroundings, but you could do it with color, `font-weight`, a `border-bottom: 1px dashed #eee`, or whatever else you want.  

It’s flexible—_and_ semantic! :-)
